dtfs-central-api/src/v1/controllers/utilisation-report-service/fee-record-correction/post-fee-record-correction.controller/helpers.ts

```diff
@@ -65,10 +63,18 @@ export const sendFeeRecordCorrectionRequestEmails = async (
   reasons: RecordCorrectionReason[],
   reportPeriod: ReportPeriod,
   exporter: string,
-  bankId: string,
   requestedByUserEmail: string,
+  recipient: string,
```

Comment:
Please can we call this `paymentOfficerTeamName`, I think the responsibility of knowing that the recipient should be the payment officer team name should belong to this function and not the controller calling this function

Reply:
done